ApplicationLibCode/ProjectDataModel/Summary/RimSummaryMultiPlot.cpp

```diff
@@ -1231,57 +1220,84 @@ void RimSummaryMultiPlot::duplicate()
 //--------------------------------------------------------------------------------------------------
 void RimSummaryMultiPlot::analyzePlotsAndAdjustAppearanceSettings()
 {
-    RiaSummaryAddressAnalyzer analyzer;
-
-    for ( auto p : summaryPlots() )
+    if ( m_autoAdjustAppearance )
     {
-        auto addresses = RimSummaryAddressModifier::createEclipseSummaryAddress( p );
-        analyzer.appendAddresses( addresses );
-    }
+        // Required to sync axis ranges before computing the auto scale
+        syncAxisRanges();
 
-    bool hasOnlyOneQuantity = analyzer.isSingleQuantityIgnoreHistory();
+        RiaSummaryAddressAnalyzer analyzer;
 
-    for ( auto p : summaryPlots() )
-    {
-        auto timeAxisProp = p->timeAxisProperties();
+        for ( auto p : summaryPlots() )
+        {
+            auto addresses = RimSummaryAddressModifier::createEclipseSummaryAddress( p );
+            analyzer.appendAddresses( addresses );
+        }
 
-        if ( columnCount() < 3 )
-            timeAxisProp->setAutoValueForMajorTickmarkCount( RimPlotAxisProperties::LegendTickmarkCount::TICKMARK_DEFAULT );
-        else
-            timeAxisProp->setAutoValueForMajorTickmarkCount( RimPlotAxisProperties::LegendTickmarkCount::TICKMARK_FEW );
+        bool canShowOneAxisTitlePerRow = analyzer.isSingleQuantityIgnoreHistory() &&
+                                         ( m_axisRangeAggregation() != AxisRangeAggregation::NONE );
 
-        for ( RimPlotAxisPropertiesInterface* axisInterface : p->plotYAxes() )
+        for ( auto p : summaryPlots() )
         {
-            auto axisProp = dynamic_cast<RimPlotAxisProperties*>( axisInterface );
-
-            if ( !axisProp ) continue;
+            auto timeAxisProp = p->timeAxisProperties();
 
-            if ( rowsPerPage() == 1 )
-                axisProp->setAutoValueForMajorTickmarkCount(
-                    RimPlotAxisPropertiesInterface::LegendTickmarkCount::TICKMARK_DEFAULT );
+            if ( columnCount() < 3 )
+                timeAxisProp->setAutoValueForMajorTickmarkCount(
+                    RimPlotAxisProperties::LegendTickmarkCount::TICKMARK_DEFAULT );
             else
-                axisProp->setAutoValueForMajorTickmarkCount(
-                    RimPlotAxisPropertiesInterface::LegendTickmarkCount::TICKMARK_FEW );
+                timeAxisProp->setAutoValueForMajorTickmarkCount( RimPlotAxisProperties::LegendTickmarkCount::TICKMARK_FEW );
```

Comment:
Maybe introduce a variable here since?
`
auto tickMarkCount =  ( columnCount() < 3 ) ? RimPlotAxisProperties::LegendTickmarkCount::TICKMARK_DEFAULT ) : RimPlotAxisProperties::LegendTickmarkCount::TICKMARK_FEW );
timeAxisProp->setAutoValueForMajorTickmarkCount( tickMarkCount );
`